transport.go

```diff
@@ -21,7 +21,7 @@ func (t *TailscaleCaddyTransport) UnmarshalCaddyfile(d *caddyfile.Dispenser) err
 func (t *TailscaleCaddyTransport) Provision(context caddy.Context) error {
 	t.logger = context.Logger()
 
-	s, err := getServer("", "caddy-tsnet-client:80")
+	s, err := getServer(caddy.Context{}, "", "caddy-tsnet-client:80")
```

Comment:
Why a fresh `caddy.Context` rather than passing the context from the argument? 🤔

Reply:
Oh cause I'm a dummy :) Thanks :sweat_smile: